DSharpPlus/Entities/Guild/DiscordGuild.cs

```diff
@@ -740,7 +751,23 @@ public async Task<IReadOnlyList<DiscordScheduledGuildEvent>> GetEventsAsync(bool
     /// <param name="limit">How many users to fetch.</param>
     /// <param name="after">Fetch users after this id. Mutually exclusive with before</param>
     /// <param name="before">Fetch users before this id. Mutually exclusive with after</param>
-    public async IAsyncEnumerable<DiscordUser> GetEventUsersAsync(DiscordScheduledGuildEvent guildEvent, int limit = 100, ulong? after = null, ulong? before = null)
+    public IAsyncEnumerable<DiscordUser> GetEventUsersAsync
+    (
+        DiscordScheduledGuildEvent guildEvent,
+        int limit = 100,
+        ulong? after = null,
+        ulong? before = null
+    )
+        => GetEventUsersAsync(guildEvent.Id, limit, after, before);

```

Comment:
It feels like something here should be inlined

Reply:
The `=>` part or everything? The line is pretty long